sql/src/test/groovy/io/crate/planner/statement/CopyToPlannerTest.java

```diff
@@ -56,8 +57,8 @@
 
     @Before
     public void prepare() throws IOException {
-        e = SQLExecutor.builder(clusterService)
-            .addDocTable(TableDefinitions.USER_TABLE_INFO)
+        e = SQLExecutor.builder(clusterService, 2, RandomizedTest.getRandom())
```

Comment:
just curious: any specific reason to use 2 nodes now?
as you changed that at a couple of suites: would it make sense to use 2 nodes by default and change tests which requires only 1 node to be concrete?

Reply:
Because many tests asserted to have `Merge` plans - this is one case where the clusterState and routing were out of sync. Our custom created routing faked 2 nodes, but the cluster state only contained 1 node.

I think the majority still has 1 node, so i think defaulting to 1 is fine.